matter_server/client/models/node.py

```diff
@@ -155,91 +173,64 @@ def set_attribute_value(self, attribute_path: str, attribute_value: Any) -> None
-            )
-        # lookup device types from node data
-        for endpoint in self.endpoints.values():
-            # get DeviceTypeList Attribute on the Descriptor cluster
-            cluster = endpoint.get_cluster(Clusters.Descriptor)
-            if not cluster:
-                LOGGER.debug(
-                    "No Descriptor cluster found on endpoint %s, Node %s",
-                    endpoint.endpoint_id,
-                    endpoint.node.node_id,
-                )
-                continue
-
-            for dev_info in cluster.deviceTypeList:  # type: ignore[unreachable]
-                device_type = DEVICE_TYPES.get(dev_info.type)
-                if device_type is None:
-                    LOGGER.debug("Found unknown device type %s", dev_info)
-                    continue
-
-                instance: MatterDeviceTypeInstance[Any] = MatterDeviceTypeInstance(
-                    self, device_type, endpoint, dev_info.revision
-                )
-                if device_type is RootNode:
-                    self.root_device_type_instance = instance
-                elif device_type is Aggregator:
-                    self.aggregator_device_type_instance = instance
-                else:
-                    self.device_type_instances.append(instance)
-
-        if self.root_device_type_instance is None:
-            raise ValueError("No root device found")
-
-        # parse node devices
-        self.node_devices = []
-        if self.aggregator_device_type_instance:
-            for instance in self.device_type_instances:
-                if instance.device_type == BridgedDevice:
-                    self.node_devices.append(MatterBridgedNodeDevice(instance))
-        else:
-            self.node_devices.append(MatterNodeDevice(self))
-
-    def update_attribute(self, attribute_path: str, new_value: Any) -> None:
-        """Handle Attribute value update."""
-        endpoint_id = int(attribute_path.split("/")[0])
-        self.endpoints[endpoint_id].set_attribute_value(attribute_path, new_value)
-
     @property
     def node_id(self) -> int:
         """Return Node ID."""
         return self.node_data.node_id
 
+    @property
+    def name(self) -> str | None:
+        """Return friendly name for this node."""
+        if info := self.device_info:
+            return info.nodeLabel
+        return None
+
     @property
     def available(self) -> bool:
         """Return availability of the node."""
         return self.node_data.available
 
+    @property
+    def device_info(self) -> Clusters.BasicInformation:
+        """
+        Return device info for this Node.
+
+        Returns BasicInformation from the Node itself (endpoint 0).
+        """
+        return self.get_cluster(0, Clusters.BasicInformation)
+
+    @property
+    def is_bridge_device(self) -> bool:
+        """Return if this Node is a Bridge/Aggregator device."""
+        return any(Aggregator in x.device_types for x in self.endpoints.values())
```

Comment:
Shouldn't that always be endpoint 0 ?

Reply:
I saw it at endpoint 1 on the Hue bridge